build_tfm.py

```diff
@@ -567,6 +567,9 @@ def _build_target(tgt, cmake_build_dir, args):
 
         _copy_tfm_ns_files(cmake_build_dir, tgt[0])
 
+    if args.commit:
+        _commit_changes(MBED_TF_M_PATH)
```

Comment:
What commit message does this make? Did we need to add this here? Should we remove the other call to `_commit_changes(MBED_TF_M_PATH)` to avoid two commits when one combined one is best?

Reply:
There are indeed two commits. Instead of a unified "Upgrade TF-M for such and such platform", we have a point in history where the binary and the supporting files are out of sync. Also, there are `"` in the commit messages themselves...

```
ec76f07b91e7 (HEAD) "Updated directory platform/FEATURE_EXPERIMENTAL_API/FEATURE_PSA/TARGET_TFM/TARGET_TFM_LATEST"
ddb94b8ee8ae "Updated secure binaries for [('ARM_MUSCA_B1', 'GNUARM')]"
```